src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/DomainEventEntity.kt

```diff
@@ -40,4 +44,11 @@ enum class DomainEventType {
   APPROVED_PREMISES_BOOKING_CANCELLED,
   APPROVED_PREMISES_BOOKING_CHANGED,
   APPROVED_PREMISES_APPLICATION_WITHDRAWN,
+  APPROVED_PREMISES_INFORMATION_REQUEST,
 }
+
+data class TimelineEvent(
```

Comment:
This might fit better in the models directory, as it should sit seperately from Domain Events